### PR TITLE
logic for getting one event

### PR DESCRIPTION
**Pull request by RodrigoACG**
## Type of Change
- [x] New Feature
- [ ] Debugging
- [ ] Refactor
- [ ] Documentation

## PR Issue link if applicable
Refer to KAN32

## Describe Changes Made
In this pull request I am accessing only one event. 

## PR Checklist
- [x] Added Reviewer
- [ ] Followed TDD, And Test are Passing

### CODE REVIEW

File: api/urls.py [hunk 1]
```diff
@@ -7,5 +7,6 @@
     path('api/users/<int:user_id>/events/<int:event_id>/join', views.join_event, name='join_event'),
     path('api/users/<int:user_id>/events/<int:event_id>/leave', views.leave_event, name='leave_event'),
     path('api/users/<int:user_id>/events', views.get_user_events, name='user_events'),
+    path('api/users/<int:user_id>/events/<int:event_id>', views.get_one_event, name='one_event'),
```
Comment (GBowman1) [inline api/urls.py:10]: event show routing looks great

File: api/views.py [hunk 2]
```diff
@@ -147,4 +147,35 @@ def get_user_events(request, user_id):
...
+    return JsonResponse({'error': 'Invalid request method'}, status=405)
+
+@csrf_exempt
+def get_one_event(request, user_id, event_id):
+    if request.method == 'GET':
+        user = get_object_or_404(User, id=user_id)
+
+        event = get_object_or_404(Event, id=event_id)
+
+        e_1 = Event.objects.filter(owner=user, id=event.id)
```
Comment (GBowman1) [inline api/views.py:159]: You figured out the filter thats awesome

File: api/views.py [hunk 3]
```diff
@@ -147,4 +147,35 @@ def get_user_events(request, user_id):
...
+                        'venue_name': event.venue_name,
+                        'date_time': event.date_time,
+                        'artist': event.artist,
+                        'location': event.location,
+                        'spotify_artist_id': event.spotify_artist_id,
+                        'ticketmaster_event_id': event.ticketmaster_event_id,
+                        'owner': user.id
+                    }
+                    for event in e_1
+                ]
```
Comment (GBowman1) [inline api/views.py:178]: Json structure looks good

**Review by GBowman1 — APPROVED** [review]

File: api/views.py (hunk 2 above)
Comment (zach-bergman) [reply api/views.py:159]: Nice!

**Review by zach-bergman — APPROVED** [review]

**Pull request merged**